include/grpcpp/security/credentials_impl.h

```diff
@@ -250,6 +251,10 @@ class MetadataCredentialsPlugin {
       grpc::string_ref service_url, grpc::string_ref method_name,
       const grpc::AuthContext& channel_auth_context,
       std::multimap<grpc::string, grpc::string>* metadata) = 0;
+
+  virtual std::string DebugString() const {
```

Comment:
I forgot to change this to grpc::string. Will address it once I've confirmed that all tests have passed